dev_docs/tutorials/development_windows.mdx

```diff
@@ -0,0 +1,147 @@
+---
+id: kibDevTutorialSetupWindowsDevWSL
+slug: /kibana-dev-docs/tutorial/setup-windows-development-wsl
+title: WSL on Windows Development
+summary: Learn how to setup a Windows development environment using WSL
+date: 2022-06-16
+tags: ['kibana', 'onboarding', 'setup', 'windows', 'development', 'wsl']
+---
+
+
+# Overview
+
+Development on Windows is recommended through WSL2.  WSL lets users run a Linux environment on Windows, providing a supported development environment for Kibana.
+
+<DocCallOut title="Prerequisites">
+  
+  ### Basic Experience
+  For a basic WSL usage you must be running **Windows 10 version 2004 and higher (Build 19041 and higher) or Windows 11**. Note that in a basic setup you won't be able to use 
+  the browser GUI to visually run and/or debug functional tests. It's highly advisable to make sure you install the prerequisites for a default usage listed below.
+  
+  **Note:** For a normal development cycle we really advise to use the default experience setup within Windows 11. However if you have an explicit need for using a browser GUI 
+  while using Windows 10, please follow each step of the guide below applicable for the Basic Experience setup and then check our <DocLink id="kibDevTutorialSetupWindowsDevWSL" section="short-term-workaround-for-windows-10-gui-support" text="short term workaround for Windows 10 GUI support" />.
+  
+  ### Default Experience
+  For a default experience and a completion of the entire guide which will make you able to run GUI apps you must be running **Windows 11 (Build 22000 and higher)**.
+  In addition to that, installing a **vGPU driver** is also required. Please choose the one compatible with your hardware: 
+
+  - [Intel GPU driver for WSL](https://www.intel.com/content/www/us/en/download/19344/intel-graphics-windows-dch-drivers.html?)
+  - [AMD GPU driver for WSL](https://www.amd.com/en/support/kb/release-notes/rn-rad-win-wsl-support)
+  - [NVIDIA GPU driver for WSL](https://developer.nvidia.com/cuda/wsl)
```

Comment:
It seems the nvidia drivers are not necessary, the vGPU drivers are described by nvidia as necessary for ML and CUDA tasks, but that standard GPU support works in WSL:

> Developers can now leverage the NVIDIA software stack on Microsoft Windows WSL environment using the NVIDIA drivers available today.
> The NVIDIA Windows GeForce or Quadro production (x86) driver that NVIDIA offers comes with CUDA and DirectML support for WSL and can be downloaded from below.

https://developer.nvidia.com/cuda/wsl

Reply:
Updated the guide to include this